sshc.go

```diff
@@ -197,7 +198,24 @@ func (c *Config) DialWithConfig() (*ssh.Client, error) {
 		}
 		fmt.Println("")
 	}
-	auth = append(auth, ssh.PublicKeys(signer))
+
+	if sshAuthSockExists() {
```

Comment:
I think we need to check `ForwardAgent` too.

```golang
if c.Get(host, "ForwardAgent") == "yes" && sshAuthSockExists() {
// ...
}
```

Reply:
I agree, need to check. I fixed in https://github.com/k1LoW/sshc/pull/6/commits/e9ad09a12781822de6c29f15716268a8362d1736 and https://github.com/k1LoW/sshc/pull/6/commits/c2ed0d3621dda5b21af9cf3c18f736a81c62c6b5 !